src/components/OwnerHome/index.js

```diff
@@ -41,6 +46,49 @@ export default class OwnerHome extends Component {
     this.setState({ addSlots: !this.state.addSlots });
   };
 
+  componentDidMount() {
+    const { user } = this.state;
+    const userId =
+      loadState('userAuthenticationDetails') &&
+      loadState('userAuthenticationDetails').userId;
+      
+    axios
+      .get(
+        `https://safeslot-backend.herokuapp.com/api/users/${userId}?filter={"include": "stores"}`
+      )
+      .then(res => {
+        const user = res.data;
+        this.setState({ user });
```

Comment:
Has this been tested? Unless you've bound the `this` (https://javascript.info/bind), because you're in a promise, the `this` has changed.